apis/messages.es6.js

```diff
@@ -50,6 +50,7 @@ export default class Messages extends BaseAPI {
             const constructor = CONSTRUCTORS[data.kind];
             const thing = new constructor(data.data).toJSON();
             if (constructor === CONSTRUCTORS.t4 &&
+                !Array.isArray(thing.replies) &&
```

Comment:
yeah... well the model changes replies to an empty array if there are none. the api returns it as an empty string, which is... weirder. All of the transformations to make them look the same are in the api layer now.